.buildkite/pipeline.sh

```diff
@@ -38,8 +38,22 @@ pr_and_no_code_related_changes() {
       ':(exclude)towncrier.toml'
 }
 
+# Helper that allows some variables to be injected into the pipeline YAML file.
```

Comment:
Hmm... this seems a bit ugly.

Could we just export an environment variable and it would be substituted by Buildkite's "standard" environment variable substitution https://buildkite.com/docs/agent/v3/cli-pipeline#environment-variable-substitution?

Reply:
Yeah that would be much nicer but I'm not sure if that works in arbitrary places like the docker image field?

Reply:
Cool, seems to work.